common/religion/doctrines/wc_tenets.txt

```diff
@@ -0,0 +1,45 @@
+	tenet_patron_loa = {
+		name = {
+			first_valid = {
+				triggered_desc = {
+					desc = tenet_patron_loa_name
+				}
+				desc = tenet_patron_loa_desc
+			}
```

Comment:
I think just using the vanilla bhakti trait would be easier than adding a patron god.
Just add Mrligion in `is_shown` in the triggers and it'll work just fine.